test/WebJobs.Script.Tests/ContentEndToEndTests.cs

```diff
@@ -19,9 +19,9 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Tests.Content
 {
-    public class NodeTextPlain : ContentTests
+    public class NodeTextPlain : ContentTests<NodeContentFixture>
```

Comment:
With xUnit, unless you're using collection fixtures, you'd get one fixture instance per class (reused on all tests on that class)